graphql/tests/commitQueue/results.json

```diff
@@ -37,6 +37,14 @@
           }
         ]
       }
+    },
+    {
+      "query_file": "remove-patch-from-commit-queue.graphql",
+      "result": {
+        "data": {
+          "removePatchFromCommitQueue": true
```

Comment:
can we return the id of the item that was removed instead of a boolean?

Reply:
Would the id be useful since we already pass it in to the function?

Reply:
it's consistent with the return values of the rest of our update/create actions. but now seeing that `CommitQueueRemoveItem` itself returns a boolean.